ecs-cli/modules/cli/local/project/project.go

```diff
@@ -0,0 +1,214 @@
+// Copyright 2015-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+// Package localproject defines LocalProject interface and implements them on localProject
+
+package localproject
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/local/converter"
+	ecsclient "github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/ecs"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/clients/aws/secretsmanager"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/commands/flags"
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/config"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/urfave/cli"
+)
+
+const (
+	LocalOutDefaultFileName = "./docker-compose.local.yml"
+	LocalOutFileMode        = os.FileMode(0644) // Owner=read/write, Other=readonly
+	LocalInFileName         = "./task-definition.json"
```

Comment:
dumb question: Do these paths work for Windows? Or do we need to do some [path.Join](https://golang.org/pkg/path/#Join) shenanigans?

Reply:
Ooh, good question. @allisaurus ?

Reply:
Hmm anecdotally, I have not run into too many issues using `/` (from terminals). But to be safe I'd prefer if we used `os.PathSeparater` as we do in [creds_writer.go](https://github.com/allisaurus/amazon-ecs-cli/blob/master/ecs-cli/modules/utils/regcredio/creds_writers.go#L62)

Reply:
Was there a reason why we didn't address this comment? :p

Reply:
Which, allie's comment? Probably bc I'm not technically back in the office until next Monday ;)